engine/DJI_E305.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!--bldc_engine name="DJI E305"-->
+  <brushless_dc_motor name="DJI E305">
+  <power unit="WATTS"> 320 </power>
+  <velocityconstant> 960 </velocityconstant>
+  <coilresistance>0.117</coilresistance>
+  <noloadcurrent>0.45</noloadcurrent>
+  <maxvolts> 14.63 </maxvolts>
+  <deceleration_factor> 5</deceleration_factor>
```

Comment:
Unless I'm mistaken `deceleration_factor` is not used by `FGBrushlessDCMotor`.

Reply:
resolved with commit corrections